### PR TITLE
Call city implementation

### PR DESCRIPTION
**Pull request by kailamjeter**
## Description

Addresses: CTA#2: Display contact info to call the city or water system 

Also fixes bug introduced in https://github.com/BlueConduit/open-data-platform/pull/157 which looks at the parcel city field instead of the computed property for the lslr section. 

### New

- Contact your city section

## Testing and Reviewing

Tested locally and in sandbox: 

[Toledo](https://kailajeter.leadout-sandbox.blueconduit.com/scorecard/address/41.724295,-83.552465) 
[New Orleans](https://kailajeter.leadout-sandbox.blueconduit.com/scorecard/postcode/29.960244,-90.073635)
[Richmond](https://kailajeter.leadout-sandbox.blueconduit.com/scorecard/postcode/37.579738,-77.53986)

### CODE REVIEW

File: client/src/components/ContactCitySection.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import { defineComponent, PropType } from 'vue';
+import { ScorecardMessages } from '../assets/messages/scorecard_messages';
+
+interface CityInformation {
+  name: City;
+  website: string;
+  phoneNumber: string;
+}
+
+const CITY_INFO: CityInformation[] = [
```
Comment (beekley) [inline client/src/components/ContactCitySection.vue:31]: Did you consider using a map here instead of a list (e.g. `Map<City, CityInformation>`)? So we don't have to iterate through the list to look it up. 

(This is a really a nit because iterating through a list of 3 objects is basically free).

File: client/src/components/ContactCitySection.vue [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  </div>
+</template>
+
+<script lang='ts'>
+
+import { City } from '../model/states/model/geo_data';
+import { defineComponent, PropType } from 'vue';
+import { ScorecardMessages } from '../assets/messages/scorecard_messages';
+
+interface CityInformation {
```
Comment (beekley) [inline client/src/components/ContactCitySection.vue:25]: nit: I'd rather name this something more specific to its purpose, such as `ContactInfo`.

File: client/src/components/ContactCitySection.vue (hunk 1 above)
Comment (beekley) [inline client/src/components/ContactCitySection.vue:31]: Also, do you know if there's any intention of getting this kind of data for more cities? If so, let's add a TODO to move this to a DB somewhere once the list gets longer than a few elements.

File: client/src/components/NationwideMap.vue [hunk 3]
```diff
@@ -99,7 +99,7 @@ export default defineComponent({
     center: {
       // There is no constructor function for a Tuple, so use object here and
       // cast to PropType of a tuple.
-      // See https://vuejs.org/guide/typescript/options-api.html#typing-component-props.
+      // See https://vuejsForg/guide/typescript/options-api.html#typing-component-props.
```
Comment (beekley) [inline client/src/components/NationwideMap.vue:102]: Is this change intentional?

File: client/src/views/ScorecardView.vue [hunk 4]
```diff
@@ -42,6 +42,11 @@
         </div>
       </div>
 
+      <ContactCitySection
```
Comment (beekley) [inline client/src/views/ScorecardView.vue:45]: Is there a reason this isn't in the section above, which has the other CTAs?

**Review by beekley — APPROVED** [review]
Left a few minor questions/comments.

File: client/src/components/ContactCitySection.vue (hunk 1 above)
Comment (kailamjeter) [reply client/src/components/ContactCitySection.vue:31]: I did, but have been hating javascript maps so was trying to avoid it (the initialization of the map to me is just so excessive). I'll change it though it is just a better way to do it.

File: client/src/components/NationwideMap.vue (hunk 3 above)
Comment (kailamjeter) [reply client/src/components/NationwideMap.vue:102]: oh no! didn't even realize I must have been command f'ing or something. Will revert!

File: client/src/views/ScorecardView.vue (hunk 4 above)
Comment (kailamjeter) [reply client/src/views/ScorecardView.vue:45]: I tried putting it inside of that div as another 1/3 of the page element but it looked really strange: https://screenshot.googleplex.com/9Ft8xw6DsXcwL5J I think because visually it is different from those other ctas so it looks strange to me that they are grouped in a row like that (as opposed to https://screenshot.googleplex.com/6bsiMhQtph5iZzg). 

Let me know what you think!

File: client/src/components/ContactCitySection.vue (hunk 2 above)
Comment (kailamjeter) [reply client/src/components/ContactCitySection.vue:25]: Change to CityContactInfo

File: client/src/components/ContactCitySection.vue (hunk 1 above)
Comment (kailamjeter) [reply client/src/components/ContactCitySection.vue:31]: Not sure honestly, probably depends on user feedback and wouldn't happen before our fellowship is over. Left a TODO in case BC ends up adding more.
